examples/simple.yaml

```diff
@@ -1,29 +1,53 @@
-# For simple deployments
-
-armada:
+# simple deploy
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: blog-1
+data:
+  name: blog-1
```

Comment:
Consider changing the attribute in the data section to something besides `name` as `name` in the metadata section is the idiomatic key for identifying a document. If `data.name` is what a chart gets named in Helm, consider `data.chartname`.